src/bun_utils.c

```diff
@@ -89,10 +95,16 @@ open_real_file(const char *name)
 	fd = open(name, O_CREAT | O_TRUNC | O_CLOEXEC | O_RDWR, S_IRUSR | S_IWUSR);
```

Comment:
are we sure this code works for `O_TMPFILE` ? For `O_TMPFILE`, `name` should specify a directory, not a filename.